### PR TITLE
Close #15384 : New widget stat database

### LINKED ISSUE #15384: Widget: Move code with statistics of database as a widget
Move code with statistics of database as a widget

### PR DESCRIPTION
**Pull request by Hystepik**
Add of a new widget which can be added by doing a migration.

### CODE REVIEW

**Comment by Hystepik** [comment]
image
image
There is the widget state board which replaces the state board that was hard coded in the index

File: htdocs/core/boxes/box_dolibarr_state_board.php [hunk 1]
```diff
@@ -0,0 +1,335 @@
...
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ *	\file       htdocs/core/boxes/box_dolibarr_state_board.php
+ *	\ingroup
+ *	\brief      Module Dolibarr state base
+ */
+
+use Symfony\Component\VarDumper\VarDumper;
```
Comment (eldy) [inline htdocs/core/boxes/box_dolibarr_state_board.php:27]: What is this ? Probably automatically added by IDE ?

File: htdocs/install/mysql/migration/12.0.0-13.0.0.sql [hunk 2]
```diff
@@ -550,5 +550,6 @@ INSERT INTO llx_c_socialnetworks (entity, code, label, url, icon, active) VALUES
 -- VMYSQL4.1 INSERT INTO llx_boxes_def (file, entity) SELECT  'box_funnel_of_prospection.php', 1 FROM DUAL WHERE NOT EXISTS (SELECT * FROM llx_boxes_def WHERE file = 'box_funnel_of_prospection.php' AND entity = 1);
 -- VMYSQL4.1 INSERT INTO llx_boxes_def (file, entity) SELECT  'box_customers_outstanding_bill_reached.php', 1 FROM DUAL WHERE NOT EXISTS (SELECT * FROM llx_boxes_def WHERE file = 'box_customers_outstanding_bill_reached.php' AND entity = 1);
 -- VMYSQL4.1 INSERT INTO llx_boxes_def (file, entity) SELECT  'box_scheduled_jobs.php', 1 FROM DUAL WHERE NOT EXISTS (SELECT * FROM llx_boxes_def WHERE file = 'box_scheduled_jobs.php' AND entity = 1);
+-- VMYSQL4.1 INSERT INTO llx_boxes_def (file, entity) SELECT  'box_dolibarr_state_board.php', 1 FROM DUAL WHERE NOT EXISTS (SELECT * FROM llx_boxes_def WHERE file = 'box_dolibarr_state_board.php' AND entity = 1);
```
Comment (frederic34) [inline htdocs/install/mysql/migration/12.0.0-13.0.0.sql:553]: @Hystepik can you move that in migration from 13 to 14 ?

Comment (Hystepik) [reply htdocs/install/mysql/migration/12.0.0-13.0.0.sql]: It's done

**Pull request merged**